### DISCUSSION TITLE
Unable to use normally in China.

### DISCUSSION THREAD

**Discussion by lylus** [llamafilm/tesla-http-proxy-addon#71] [Q&A]
The account created in China cannot be used and the displayed ID does not exist. The website for applying in China is as follows:
developer. tesla.cn
But even in the "Tesla HTTP Proxy" option China, the access still ends with com instead of cn. Causing the display ID to not exist.

[08:54:41] INFO: Running auth.py
[08:54:41] auth:INFO: Generating Partner Authentication Token
[08:54:41] urllib3.connectionpool:DEBUG: Starting new HTTPS connection (1): auth.tesla.com:443
[08:54:42] urllib3.connectionpool:DEBUG: https://auth.tesla.com:443 "POST /oauth2/v3/token HTTP/1.1" 400 197
[08:54:42] auth:ERROR: HTTP 400: Bad Request
[08:54:42] auth:DEBUG: {
  "error": "unauthorized_client",
  "error_description": "We don't recognize this client_id and client_secret combination",
  "referenceID": "1025732a-e074-47d2-81f3-0a16055000d2-1712019281937"
}
[08:54:42] auth:ERROR: Response did not include access token: {
  "error": "unauthorized_client",
  "error_description": "We don't recognize this client_id and client_secret combination",
  "referenceID": "1025732a-e074-47d2-81f3-0a16055000d2-1712019281937"
}
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service webui: stopping
s6-rc: info: service webui successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped

**Comment by llamafilm** [comment]
I’m going to convert this to issue because it’s a bug. #72
